cli/admin/src/main/java/io/pravega/cli/admin/controller/metadata/ControllerMetadataViewPendingEventsCommand.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.cli.admin.controller.metadata;
+
+import io.pravega.cli.admin.CommandArgs;
+import io.pravega.cli.admin.utils.ZKHelper;
+
+/**
+ * To view pending event detail for a request in a particular controller host.
```

Comment:
What kind of events are you describing here? A user may get confused, as the word "event" is used in manu contexts. Add few more lines about the context and the goal/purpose of this command (or even its usage).

Reply:
added details